src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size = 0;
+    private transient T[] elementData;
```

Comment:
what does `transient` mean and why is this field transient? :)

Reply:
How I understand this term, This field is like "temporary", won’t  save to file(won’t be part of the serialization, but I haven’t delved into this topic).